BibBuddy/app/src/androidTest/java/de/bibbuddy/NotesFragmentUITest.java

```diff
@@ -19,191 +22,173 @@
-    onView(withId(nF.getActivity().getSupportFragmentManager().findFragmentById(R.id.fragment_picture_note_editor).getId())).check(matches(isDisplayed()));
-    *//*
-
-
+  static NotesFragment nF;
+  private static View itemView;
+  private final String exampleText = "text";
+  private String modDate;
+  private String name;
+  private int idText;
+  private int idVoice;
+  private int idPicture;
+
+  @Before
+  public void init() {
+    ActivityScenario scenario = ActivityScenario.launch(MainActivity.class);
+    onView(withId(R.id.navigation_notes)).perform(click());
+    onView(withId(R.id.fragment_notes)).check(matches(isDisplayed()));
+    ActivityScenario activityScenario = scenario.onActivity(
+        (ActivityScenario.ActivityAction<MainActivity>) activity -> {
+          nF = (NotesFragment) activity.getSupportFragmentManager().getFragments().get(0);
+          NotesFragment.notes.clear();
+          if (NotesFragment.notes.size() == 0) {
+            Note textNote =
+                new Note(exampleText, 0, exampleText);
+            modDate = String.valueOf(textNote.getModDate());
+            name = textNote.getName();
+            NoteTextItem noteTextItem = new NoteTextItem(modDate, name, textNote.getId());
+            NotesFragment.notes.add(noteTextItem);
+          }
+        });
+
+    onView(ViewMatchers.withId(R.id.recyclerView))
+        .perform(RecyclerViewActions.scrollTo(hasDescendant(withText(exampleText))));
+    RecyclerView recyclerView =
+        Objects.requireNonNull(nF.getView()).findViewById(R.id.recyclerView);
+    itemView = recyclerView.getChildAt(0);
+    idText = itemView.getId();
+  }
+
+  @Test
+  public void NotesFragmentDisplay_Test() {
+    onView(withId(R.id.fragment_notes)).check(matches(isDisplayed()));
+  }
+
+  // TODO: Needs to be adjusted as soon as more note types are added
+  @Test
+  public void RecyclerViewListDisplay_Test() {
+    ViewActions.closeSoftKeyboard();
+    onView(withId(idText)).check(matches(isDisplayed()));
+    onView(withId(R.id.noteModDate)).check(matches(isDisplayed()));
+    onView(withId(R.id.noteName)).check(matches(isDisplayed()));
+    onView(withId(R.id.noteType)).check(matches(isDisplayed()));
+    new DrawableMatcher(R.drawable.document)
+        .matchesSafely(itemView);
+    // onView(withId(R.id.noteType)).check(matches(withId(R.drawable.microphone));
+    // onView(withId(R.id.noteType)).check(matches(withId(R.drawable.picture)));
+  }
+
+  // TODO: Needs to be adjusted as soon as more note types are added
+  @Test
+  public void OpenEditorOnItemClick_Test() {
+    ViewActions.closeSoftKeyboard();
+    onView(withId(idText)).perform(click());
+    onView(withId(R.id.fragment_text_note_editor)).check(matches(isDisplayed()));
+      /*
+      onView(withId(id2)).perform(click());
+      onView(withId(nF.getActivity().getSupportFragmentManager().findFragmentById(R.id.fragment_voice_note_editor).getId())).check(matches(isDisplayed()));
```

Comment:
This may resolve your java checkstyle error:
```
onView(withId(nF.getActivity().getSupportFragmentManager()
     .findFragmentById(R.id.fragment_voice_note_editor).getId()))
     .check(matches(isDisplayed()));
```

Reply:
I removed that comment.